pkg/utils/kubernetes/utils.go

```diff
@@ -0,0 +1,130 @@
+package kubernetes
```

Comment:
These two probably fit better in folder/package called finalizer

Reply:
The idea of this package is to have kubernetes specific helpers, similar to what we have for openshift.

Reply:
Right, but it's a lot of code and if we now added a new utility method in this file, it would be very hard to find. That's why I was thinking either a package called finalizer under kubernetes, or if not, at least rename these two files to finalizer.go and finalizer_test,go